lib/themes.dart

```diff
@@ -437,7 +437,80 @@ class ThoriumTheme extends BaseTheme {
   AppIconEnum appIcon = AppIconEnum.THORIUM;
 }
 
-enum AppIconEnum { NATRIUM, TITANIUM, INDIUM, NEPTUNIUM, THORIUM }
```

Comment:
I missed this actually, AppIconEnum is for on iOS devices the app icon changes with the theme, but since there is no icon for your theme you should just use the NATRIUM (default) icon.